src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -62,6 +62,12 @@ internal class BuildCommand : BaseCommand
             Description = "The destination directory.")]
         public string DestinationDir { get; set; }
 
+        [Option(
+            "--bp-layers-dir <dir>",
+            CommandOptionType.SingleValue,
+            Description = "Path to the layers directory, if invoked from within a Buildpack.")]
+        public string BuildpackLayersDir { get; set; }
```

Comment:
I do not see changes related to this property in this class. I guess this is kind of Todo for future?

Reply:
this is actually a temp change that should have been reverted - thanks!